nilearn/reporting/data/html/report_body_template.html

```diff
@@ -24,11 +24,6 @@
       {{endif}}
     </div>
   </div>
-  <div class="pure-u-1 pure-u-md-1-3">
-    {{if description}}
-    <p>{{description}}</p>
-    {{endif}}
-  </div>
```

Comment:
If we don't define `description` during instantiation, we can't expect it here

Reply:
But it should nevertheless show up after fitting because the `report_body_template_surfacemasker.html` template has a field for it.

Actually let me confirm that

Reply:
Yes it does 👍🏽